lab2.js

```diff
@@ -85,13 +125,30 @@ var hello = {
 // speak, and method (that you'll place on the prototype) called
 // sayHello.
 
-function SentientBeing () {
+function SentientBeing(homePlanet, lng) {
+  this.homePlanet = homePlanet;
+  this.lng = lng;
   // TODO: specify a home planet and a language
   // you'll need to add parameters to this constructor
 }
 
 // sb is a SentientBeing object
-function sayHello (sb) {
+SentientBeing.prototype.sayHello = function(sb) {
+  switch (sb.lng) {
+
+    case 'klingon':
+      console.log(hello[this.lng]);
+      return hello.klingon;
+
+    case 'romulan':
+      console.log(hello[this.lng]);
+      return hello.romulan;
+
+    default:
+      console.log(hello[this.lng]);
+      return hello['federation standard'];
```

Comment:
you don't really need the switch case here. If
sb.lng 
is using the same property names as what's in the hello var, you can do this:
console.log(hello[this.lng]);
return hello[sb.lng];